packages/widget/src/controllers/wallet-manager/token-balance.ts

```diff
@@ -100,4 +119,25 @@ export class TokenBalanceController implements ReactiveController {
       this.host.requestUpdate();
     }.bind(this)();
   };
+
+  suscribeSubstrateBalanceUpdate = (resource: SubstrateResource): void => {
+    const { substrateProvider: apiPromise, accounts } = this.walletContext.value
+      ?.substrateWallet as SubstrateWallet;
+
+    const accountAddress = accounts![0].address;
+
+    void async function (this: TokenBalanceController) {
+      const balance = await getAssetBalance(
+        apiPromise as ApiPromise,
+        (resource as unknown as { assetID: number }).assetID,
```

Comment:
```suggestion
        (resource as SubstrateResource as { assetID: number }).assetID,
```
or whatever the name of type is

Reply:
linter is removing and replacing this for unknown. which is fine. This [PR](https://github.com/sygmaprotocol/sygma-sdk/pull/357) allows us to not type cast any special stuff

Reply:
solved